app/src/main/report/README.md

```diff
@@ -0,0 +1,12 @@
+# Montagu Reporting portal
+This portal allows funders (and anyone else) to access the reports in the Montagu Reports Orderly store.
+
+# Development
+1. Run `npm install` to get dependencies and `npm install webpack --global` to install webpack
+2. Run `webpack` to build, or `webpack --watch` to continuously monitor files and
+rebuild as needed.
+3. You will need to run both the Montagu API and the Montagu Reporting API using a shared RSA keypair. 
+Each of these contain a script called `runAPIForWebappDev` and running both of these in either order will set everything up. 
+Note that these scripts need to be run as `sudo` and will generate a keypair in `/etc/montagu/api/token_key.`
```

Comment:
Once you are using the cert tool to generate the keypair (over in the API repo) then you could actually directly map `/etc/montagu/api/token_key/` as the dir (volume) to output to. Then you wouldn't need sudo to run this (just membership of the docker group).